source/admin_manual/list_of_events.rst

```diff
@@ -1691,9 +1691,10 @@ Events emitted by the metacache library.
 
    :field metacache_status:
      Status of the refresh operation:
-       * refresh_changed: Bundles were listed in storage. New bundles were
-         found and downloaded.
-       * refresh_unchanged: Bundles were listed in storage, but no new changes
+       * refreshed_changed: Bundles were listed in storage. New bundles were
+         found and downloaded. This is used also when an existing mailbox is
+	 first downloaded to metacache.
```

Comment:
I think this is missing a space or two here.

Reply:
Oh, my editor added a tab there. Fixed.